static/js/analytics.js

```diff
@@ -28,13 +28,29 @@ const composeUserChoicePage = (users) => {
     .sort((a, b) => a.name.localeCompare(b.name))
     .forEach(({ name, id }) => {
       pageHtml += `<a href='${baseUrl}/analytics/${id}'>
-                    <ul>${name}</ul>
+                    <div class="user-preview">${name}</div>
                    </a>`;
     });
   pageHtml += "</div>";
   return `<div class='analitics-wrapper'>${pageHtml}</div>`;
 };
 
+const composeDatePicker = () =>
```

Comment:
Just add this to the HTML directly instead of generating it on the JS side, it doesn't change. Make it default hidden and you can still use the condition at line 291 to make it visible if you want to have this only on successful responses.

Reply:
Done